src/main/scala/tutorial/CreateNDVIPng.scala

```diff
@@ -21,11 +21,7 @@ object CreateNDVIPng {
       // and perform the NDVI calculation.
       println("Performing NDVI calculation...")
       tile.combineDouble(0, 1) { (r: Double, ir: Double) =>
```

Comment:
It looks like we changed the band index meaning. so I believe this `combineDouble(0, 1)` needs changing.